src/test/java/DarkModeSwitcherTest.java

```diff
@@ -0,0 +1,121 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.Color;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class DarkModeSwitcherTest {
+
+    private WebDriver driver;
+
+    @BeforeSuite
+    public void setupDriverManager() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeTest
```

Comment:
Please, change to `@BeforeMethod`. So this function will execute before each method in `DarkModeSwitcherTest` class. In current version `@BeforeTest` executes only once before all methods in class and the second test method can't open login page because user is already signed in.